internal/compiler/passes/lower_states.rs

```diff
@@ -182,22 +189,48 @@ fn compute_state_property_name(root_element: &ElementRc) -> String {
 enum ExpressionForProperty {
     TwoWayBinding,
     Expression(Expression),
+    /// Workaround: the expression can't be represented with the current data structure, so make it an error for now.
+    InvaliBecauseOfIssue1461,
```

Comment:
I think that there’s a typo :-) Invali += d

Reply:
Fixed.